.ebextensions/04_container_commands.config

```diff
@@ -1,8 +1,8 @@
 container_commands:
-  00_reload_nginx:
-    command: "sudo service nginx reload"

```

Comment:
This was causing breaks with deployment process during scaling and normal deployment.